src/lib.rs

```diff
@@ -3,29 +3,22 @@ This crate provides convenience methods for encoding and decoding numbers in
 either [big-endian or little-endian order] on top of asynchronous I/O streams.
 It owes everything to the magnificent [`byteorder`] crate. This crate only
 provides a shim to [`AsyncRead`] and [`AsyncWrite`].
-
 The organization of the crate mirrors that of `byteorder`. A trait, [`ByteOrder`], specifies
 byte conversion methods for each type of number in Rust (sans numbers that have
 a platform dependent size like `usize` and `isize`). Two types, [`BigEndian`]
 and [`LittleEndian`] implement these methods. Finally, [`AsyncReadBytesExt`] and
 [`AsyncWriteBytesExt`] provide convenience methods available to all types that
 implement [`Read`] and [`Write`].
-
 An alias, [`NetworkEndian`], for [`BigEndian`] is provided to help improve
 code clarity.
-
 An additional alias, [`NativeEndian`], is provided for the endianness of the
 local platform. This is convenient when serializing data for use and
 conversions are not desired.
-
 # Examples
-
 Read unsigned 16 bit big-endian integers from an [`AsyncRead`] type:
-
 ```rust
 use std::io::Cursor;
-use tokio_byteorder::{BigEndian, AsyncReadBytesExt};
-
+use tokio_byteorder::tokio::{BigEndian, AsyncReadBytesExt};
```

Comment:
I think we should continue re-exporting these from the root of the crate, since they don't differ between the tokio and futures versions.

Reply:
Actually, this should just do:
```rust
use tokio_byteorder::prelude::*;
```

Reply:
Even in futures.rs and tokio.rs?

Reply:
Yup. The whole idea behind the prelude is that it includes all the traits. It should generally be all you need.